website/docs/r/network_isolated_v2.html.markdown

```diff
@@ -0,0 +1,120 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: vcd_network_isolated_v2"
+sidebar_current: "docs-vcd-resource-network-isolated-v2"
+description: |-
+  Provides a VMware Cloud Director Org VDC isolated Network. This can be used to create, modify, and
+  delete isolated VDC networks (backed by NSX-T or NSX-V).
+---
+
+# vcd\_network\_isolated\_v2
+
+Provides a VMware Cloud Director Org VDC isolated Network. This can be used to create, modify, and
+delete isolated VDC networks (backed by NSX-T or NSX-V).
+
+Supported in provider *v3.2+* for both NSX-T and NSX-V VDCs.
+
+## Example Usage (NSX-T backed isolated Org VDC network)
+
+```hcl
+resource "vcd_network_isolated_v2" "nsxt-backed" {
+  org  = "my-org"
+  vdc  = "my-nsxt-org-vdc"
+
+  name        = "nsxt-isolated 1"
+  description = "My isolated Org VDC network backed by NSX-T"
+
+  gateway = "1.1.1.1"
+  prefix_length = 24
+
+  static_ip_pool {
+    start_address = "1.1.1.10"
+    end_address = "1.1.1.20"
+  }
+
+  static_ip_pool {
+    start_address = "1.1.1.100"
+    end_address = "1.1.1.103"
+  }
+}
+```
+
+## Example Usage (NSX-V backed isolated Org VDC network shared with other VDCs)
+
+```hcl
+resource "vcd_network_isolated_v2" "nsxv-backed" {
+  org  = "my-org"
+  vdc  = "my-nsxv-org-vdc"
+  
+  name        = "nsxv-isolated-network"
+  description = "NSX-V isolated network"
+
+  is_shared = true
+
+  gateway = "1.1.1.1"
+  prefix_length = 24
+
+  static_ip_pool {
+    start_address = "1.1.1.10"
+    end_address = "1.1.1.20"
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `org` - (Optional) The name of organization to use, optional if defined at provider level. Useful 
+  when connected as sysadmin working across different organisations
+* `vdc` - (Optional) The name of VDC to use, optional if defined at provider level
+* `name` - (Required) A unique name for the network
+* `description` - (Optional) An optional description of the network
+* `shared` - (Optional) **NSX-V only.** Defines if this network is shared between multiple VDCs
+  in the Org.  Defaults to `false`.
+* `gateway` (Required) The gateway for this network (e.g. 192.168.1.1)
+* `prefix_length` - (Required) The prefix length for the new network (e.g. 24 for netmask 255.255.255.0).
+* `dns1` - (Optional) First DNS server to use.
+* `dns2` - (Optional) Second DNS server to use.
+* `dns_suffix` - (Optional) A FQDN for the virtual machines on this network
+* `static_ip_pool` - (Optional) A range of IPs permitted to be used as static IPs for
+  virtual machines; see [IP Pools](#ip-pools) below for details.
+
+<a id="ip-pools"></a>
+## IP Pools
+
+Static IP Pools and DHCP Pools support the following attributes:
```

Comment:
I'm confused. We support DHCP pools here? I thought not, because (quote from previous PR):

> Note. It does not include DHCP pool definition, because this depends on backing network provider (NSX-V and/org NSX-T) therefore it will be two separate resources a bit later (NSX-V and NSX-T DHCP management).

Reply:
You caught me here. I ported it from `vcd_network_imported`, but this one will support DHCP via separate resources (depending on backing VDC type `NSX-T` or `NSX-V`). Will clean up.

Reply:
Removed DHCP mention.